src/style.css

```diff
@@ -1,3 +1,83 @@
+html {
+  margin: 0 auto;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  background-color: #f5f5f5;
+  margin: 0;
+}
+
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+/* Header Styles */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 100%;
+  background-color: white;
+}
+
+.logo img {
+  display: block;
+  height: 40px;
+}
+
+/* Navigation Styles */
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list {
+  height: 60px;
```

Comment:
The height is set for the `.nav-list`, but according to the task requirements, the header height should be set for the navigation links instead.